std/typecons.d

```diff
@@ -1168,6 +1169,99 @@ template Tuple(Specs...)
 }
 
 /**
+    Allows to unpack a $(D Tuple) range to arguments when iterating using $(D foreach).
+
+    Examples:
+    ---
```

Comment:
Example in DDoc are not used anymore in new additions. Rather make some DDoc uniitest. AFter your template add some unittest with
```D
/// some description
unittest
{
   // some demo
}
```